types/react-map-gl/index.d.ts

```diff
@@ -68,30 +75,71 @@ export class StaticMap extends React.Component<StaticMapProps> {
     queryRenderedFeatures(geometry?: MapboxGL.PointLike | MapboxGL.PointLike[], parameters?: QueryRenderedFeaturesParams): Array<GeoJSON.Feature<GeoJSON.GeometryObject>>;
 }
 
-export interface InteractiveMapState {
+export interface ExtraState {
     isDragging: boolean;
     isHovering: boolean;
 }
 
-export interface MapEvent {
-    lngLat: [number, number];
-    features: Array<{}>;
+export interface PositionInput {
+  pos: [number, number];
 }
 
-export interface MjolnirEvent {
-    type: string;
-    offsetCenter: {x: number, y: number};
-    deltaX: number;
-    deltaY: number;
-    delta: number;
-    scale: number;
-    rotation: number;
-    pointerType: string;
-    metaKey: boolean;
-    rightButton: boolean;
-    stopPropagation: () => void;
-    preventDefault: () => void;
-    srcEvent: any;
+export interface MapControllerOptions {
+    onViewportChange?: (viewState: ViewState) => void;
+    onStateChange?: (state: MapState) => void;
+    eventManager?: any;
+    isInteractive: boolean;
+    scrollZoom?: boolean;
+    dragPan?: boolean;
+    dragRotate?: boolean;
+    doubleClickZoom?: boolean;
+    touchZoom?: boolean;
+    touchRotate?: boolean;
+    keyboard?: boolean;
+}
+
+export interface ViewportProps {
+  width: number;
```

Comment:
We should use 4 spaces for the indentation size.

Reply:
Good catch! Should be fixed now.